### PR TITLE
Fix: check postscript_dir instead of prescript_dir

### PR DESCRIPTION
**Pull request by narasimhan-v**
Fix the check where postscript_dir has to be checked instead of
prescript_dir.
Also spelling of postscript is fixed.

Signed-off-by: Narasimhan V <sim@linux.vnet.ibm.com>

### CODE REVIEW

File: avocado-setup.py [hunk 1]
```diff
@@ -345,11 +345,11 @@ def bootstrap(enable_kvm=False):
...
                 os.makedirs(prescript_dir)
             helper.copy_dir_file(prescript, prescript_dir)
-    if os.path.isdir(postscipt):
-        if len(os.listdir(postscipt)):
-            if not os.path.exists(prescript_dir):
-                os.makedirs(postscipt_dir)
-            helper.copy_dir_file(postscipt, postscipt_dir)
+    if os.path.isdir(postscript):
+        if len(os.listdir(postscript)):
+            if not os.path.exists(postscript_dir):
```
Comment (narasimhan-v) [inline avocado-setup.py:350]: @sathnaga This is the main fix. Others are just spelling fixes

**Comment by sathnaga** [comment]
@narasimhan-v Thanks for the fix, missed in review :-)

**Comment by narasimhan-v** [comment]
```
# ./avocado-setup.py --bootstrap
15:13:04 INFO    : Check for environment
15:13:04 INFO    : Creating temporary mux dir
15:13:04 INFO    : Creating Avocado Config
15:13:06 INFO    : Bootstrapping Avocado
15:13:06 INFO    : Cloning the repo: avocado in /root/wrapper_testing/tests/avocado
15:13:08 INFO    : Installing avocado from /root/wrapper_testing/tests/avocado
15:13:09 INFO    : Cloning the repo: avocado-misc-tests in /root/wrapper_testing/tests/tests/avocado-misc-tests
15:13:13 INFO    : copied all files from  /root/wrapper_testing/tests/config/prescript to /etc/avocado/scripts/job/pre.d/
Traceback (most recent call last):
  File "./avocado-setup.py", line 645, in <module>
    bootstrap(args.enable_kvm)
  File "./avocado-setup.py", line 352, in bootstrap
    helper.copy_dir_file(postscipt, postscipt_dir)
  File "/root/wrapper_testing/tests/lib/helper.py", line 169, in copy_dir_file
    shutil.copy2(s_file, d_file)
  File "/usr/lib64/python3.6/shutil.py", line 263, in copy2
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "/usr/lib64/python3.6/shutil.py", line 121, in copyfile
    with open(dst, 'wb') as fdst:
FileNotFoundError: [Errno 2] No such file or directory: '/etc/avocado/scripts/job/post.d/package_remove.sh'
```
```
# ./avocado-setup.py --bootstrap
15:13:57 INFO    : Check for environment
15:13:57 INFO    : Creating Avocado Config
15:13:59 INFO    : Bootstrapping Avocado
15:13:59 INFO    : Cloning the repo: avocado in /root/wrapper_testing/tests/avocado
15:14:00 INFO    : Installing avocado from /root/wrapper_testing/tests/avocado
15:14:02 INFO    : Cloning the repo: avocado-misc-tests in /root/wrapper_testing/tests/tests/avocado-misc-tests
15:14:03 INFO    : copied all files from  /root/wrapper_testing/tests/config/prescript to /etc/avocado/scripts/job/pre.d/
15:14:03 INFO    : copied all files from  /root/wrapper_testing/tests/config/postscript to /etc/avocado/scripts/job/post.d/
15:14:04 WARNING : Avocado html plugin not installed
15:14:04 INFO    : Installing optional plugin: html
15:14:05 WARNING : Avocado varianter_yaml_to_mux plugin not installed
15:14:05 INFO    : Installing optional plugin: varianter_yaml_to_mux
15:14:05 INFO    : Removing temporary mux dir
# 
```

**Comment by narasimhan-v** [comment]
@sathnaga done

**Pull request merged**